pom.xml

```diff
@@ -153,6 +154,16 @@
             <target>1.8</target>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.antlr</groupId>
+          <artifactId>antlr3-maven-plugin</artifactId>
+          <version>3.5.2</version>
```

Comment:
nit: can we use version as antlr.version property? It's already defined in this pom.xml